test/common/color/convert-color.test.ts

```diff
@@ -51,4 +51,16 @@ describe("Color Conversion Tests", () => {
     expect(theme2hex("#ff0000")).toBe("#ff0000");
     expect(theme2hex("unicorn")).toBe("unicorn");
   });
+
+  it("should convert rgb theme color to hex", () => {
+    expect(theme2hex("rgb( 255, 0, 0)")).toBe("#ff0000");
+    expect(theme2hex("rgb(0,255, 0)")).toBe("#00ff00");
+    expect(theme2hex("rgb(0, 0,255 )")).toBe("#0000ff");
+  });
+
+  it("should convert rgba theme color to hex by ignoring alpha", () => {
```

Comment:
I decided to ignore the alpha because the energy dashboard will add its own transparency to these colors, and it looks best with the colors retaining their original strength.